src/test/java/com/desiato/puresynth/repositories/UserRepositoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.desiato.puresynth.repositories;
+
+import com.desiato.puresynth.models.User;
+import com.desiato.puresynth.services.UserService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+public class UserRepositoryTest {
```

Comment:
Why do you need it?

Reply:
when everything was not working I tried to test the UserRepository, because in logs it was the initial problem of all the injection issues. but now I can remove it